examples/remote_query/mnist/webapp/src/app.jsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+import React from "react";
+import ReactDOM from "react-dom";
+import {
+  BaseFrontend,
+  LoadingScreen,
+  Instructions,
+} from "./components/core_components.jsx";
+
+import {
+  MephistoContext,
+  useMephistoRemoteQueryTask,
+  ErrorBoundary,
+} from "mephisto-task";
+
+/* ================= Application Components ================= */
+
+// Generate a random id
+function uuidv4() {
+  return "xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx".replace(/[xy]/g, function (c) {
+    var r = (Math.random() * 16) | 0,
+      v = c == "x" ? r : (r & 0x3) | 0x8;
+    return v.toString(16);
+  });
+}
+
+function RemoteQueryApp() {
+  let mephistoProps = useMephistoRemoteQueryTask({});
+
+  let {
+    blockedReason,
+    blockedExplanation,
+    taskConfig,
+    isPreview,
+    previewHtml,
+    isLoading,
+    handleSubmit,
+    remoteProcedure,
+    isOnboarding,
+    handleFatalError,
+  } = mephistoProps;
+
+  const classifyDigit = remoteProcedure("classify_digit");
```

Comment:
we say `remoteProcedure` here but `useMephistoRemoteQueryTask`, should we align on either query or procedure? unless there's some reason to have differentiation

Reply:
I can convert to `remoteProcedure` for all, this is a reasonable suggestion!